erts/doc/src/erlang.xml

```diff
@@ -52,6 +59,300 @@
   </description>
 
   <datatypes>
+    <datatype_title>Predefined datatypes</datatype_title>
+    <datatype>
+      <name name="any" n_vars="0"/>
+      <desc>
+        <p>The top type, the set of all Erlang terms.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="arity" n_vars="0"/>
+      <desc>
+        <p>The arity of a function or type.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="atom" n_vars="0"/>
+      <desc>
+        <p>An Erlang <seeguide marker="system/reference_manual:data_types#atom">atom</seeguide>.</p>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="binary" n_vars="0"/>
+      <desc>
+        <p>An Erlang <seeguide marker="system/reference_manual:data_types#bit-strings-and-binaries">binary</seeguide>,
+          i.e. a bitstring with a size divisible by 8.</p>
```

Comment:
```suggestion
          that is, a bitstring with a size divisible by 8.</p>
```

(Our guidlines for documentation says that we should avoid abbreviations such as i.e. and e.g.)